src/main/asciidoc/_chapters/hbase-default.adoc

```diff
@@ -2161,3 +2161,36 @@ The percent of region server RPC threads failed to abort RS.
 +
 .Default
 `0`
+
+
+[[hbase.master.regions.recovery.check.interval]]
+*`hbase.master.regions.recovery.check.interval`*::
++
+.Description
+
+    Regions Recovery Chore interval in milliseconds.
+    This chore keeps running at this interval to
+    find all regions with configurable max store file ref count
+    and reopens them.
+
++
+.Default
+`1200000`
+
+
+[[hbase.regions.recovery.store.file.count]]
+*`hbase.regions.recovery.store.file.count`*::
++
+.Description
+
+      Store files Ref Count threshold value considered
```

Comment:
We can say bit more detail here. Very large ref count on a file indicates that its a ref leak on that object. Such files can not get removed even after its invalidation via compaction or so.  Only way to come out of such situation is the reopen of the region